cfme/tests/test_modules_importable.py

```diff
@@ -14,7 +14,6 @@
     'cfme/utils/conf.py',  # config object that replaces the module
     'cfme/intelligence/rss.py',  # import loops
     'cfme/intelligence/chargeback/rates.py',
-    'cfme/intelligence/chargeback/assignments.py',
```

Comment:
@izapolsk I'm not sure whether this was required but I guess since this module is no longer directly imported, testing whether it can be imported is also unnecessary